packages/launcher/src/widget.tsx

```diff
@@ -68,7 +68,7 @@ export class LauncherModel extends VDomModel implements ILauncher.IModel {
     return new ArrayIterator(this._items);
   }
 
-  private _items: ILauncher.IItemOptions[] = [];
+  protected _items: ILauncher.IItemOptions[] = [];
```

Comment:
Normally the convention would be to drop the leading `_` when the attributes are not private anymore:

```suggestion
  protected items: ILauncher.IItemOptions[] = [];
```

Reply:
Ah good point, searching the code base for `protected _` returns quite a few results.

> also worth noting that there's already a function called items() in the class

In that case it indeed looks like keeping `_items` would be the safest for now :+1:

Reply:
Nice that change looks good.